app/medInria/medHomepageArea.cpp

```diff
@@ -284,33 +314,9 @@ medHomepageArea::medHomepageArea ( QWidget * parent ) : QWidget ( parent ), d (
     d->stackedWidget->addWidget(d->aboutWidget);
     d->stackedWidget->addWidget(d->pluginWidget);
     d->stackedWidget->addWidget(d->settingsWidget);
+    d->stackedWidget->addWidget(d->logWidget);
     d->stackedWidget->setCurrentIndex(0);//d->infoWidget
     d->stackedWidget->setSizePolicy(QSizePolicy::Maximum,QSizePolicy::Maximum);
-
-    //Setup homepage animations
```

Comment:
In fact, the animation had a link with the ergonomy of the homepage. The animation needed to be updated because of the new Log button. That's why i worked on it. So, after some tests, and discussions, the best was to remove it.